binsync/decompilers/ghidra/controller.py

```diff
@@ -130,8 +130,40 @@ def fill_function_header(self, func_addr, user=None, artifact=None, decompilatio
     @fill_event
     @ghidra_transaction
     def fill_struct(self, struct_name, header=True, members=True, artifact=None, **kwargs):
-        struct: Struct = artifact;
-
+        changes = True
+        struct: Struct = artifact
+        old_ghidra_struct = self._get_struct_by_name('/'+struct_name)
+        data_manager = self.ghidra.currentProgram.getDataTypeManager()
+        t = self.ghidra.currentProgram.startTransaction('add struct')
+        try:
+            handler = self.ghidra.import_module_object("ghidra.program.model.data", "DataTypeConflictHandler")
+            structType = self.ghidra.import_module_object("ghidra.program.model.data", "StructureDataType")
+            byteType = self.ghidra.import_module_object("ghidra.program.model.data", "ByteDataType")
+            ghidra_struct = structType(struct_name, 0)
+            for offset in struct.members:
+                member = struct.members[offset]
+                ghidra_struct.add(byteType.dataType, 1, member.name, "")
+                ghidra_struct.growStructure(member.size-1)
+                for dtc in ghidra_struct.getComponents():
+                    if dtc.getFieldName() == member.name:
+                        if member.type:
+                            gtype = self.typestr_to_gtype(member.type)
+                        else:
+                            gtype = self.typestr_to_gtype('undefined'+str(member.size))
```

Comment:
The naming style we are going for is `field_0`, where `0` is the offset in the struct in hex (not the size). Fix this

Reply:
That is for field name though, this is for the type definition, i'm pretty sure ghidra requires the typestr for undefined types to be undefined + field size